draft-ietf-tls-esni.md

```diff
@@ -215,14 +215,19 @@ The ECH configuration is defined by the following `ECHConfig` structure.
     struct {
         HpkeKdfId kdf_id;
         HpkeAeadId aead_id;
-    } ECHCipherSuite;
+    } HpkeSymmetricCipherSuite;
```

Comment:
Why this name change?

Reply:
It seems to capture the intent of the struct better.